bears/python/PycodestyleBear.py

```diff
@@ -35,7 +35,7 @@ def create_arguments(
         :param max_line_length:
             Limit lines to this length.
         """
-        arguments = [r"--format='%(row)d %(col)d %(code)s %(text)s'"]
+        arguments = [r"--format='%(row)d %(col)d %(code)s %(text)s"]
```

Comment:
@dracarys09 I actually tried experimenting, and put something like: 
` arguments = [r"--format='%(row)d %(col)d %(code)s %(text)s'Hello, world!"]`
The output was: `E501 line too long (213 > 79 characters)'Hello, world!`
Hence I suspected that this would be the error, though I myself thought that this has to be the closing for that initial single quote. Will see that.